styles/styles.css

```diff
@@ -647,6 +647,10 @@ main .section.narrower {
   margin: auto;
 }
 
+main .section.narrower p{
```

Comment:
How about using the `spaced` style? Would that be similar enough for your use-case?

Reply:
Ok so I believe you need this to add padding to the last p, my pages, for example https://main--sunstar--hlxsites.hlx.page/about/business-performance has multiple paragraphs, so maybe change it to `p:last-of-type`

What I was suggesting at first was to use the `narrower` style in combination with the `spaced` style, the `spaced` style also adds bottom padding to paragraphs, 2rem instead of 3rem, but that might be fine if the page you're working on looks good with that 2rem padding; that way we don't have to change the existing styles

Reply:
@dnbute @sdmcraft as discussed we needed specific bottom padding for all 'p' elements in narrower section (for any extra spaces I have used spacer block)

Reply:
The formatting seems to be a bit off .
```
p{
```

Reply:
@dnbute  Satya approved the PR. Can you please confirm at your end too so that I can go ahead with merge.